cmd/service/utils.go

```diff
@@ -31,8 +35,32 @@ func handleError(err error) {
 	}
 }
 
+func gitClone(url string) (path string, err error) {
+	tmpFile := "/tmp/mesg-templates"
+	err = os.MkdirAll(tmpFile, os.ModePerm)
+	if err != nil {
+		return
+	}
+	path, err = ioutil.TempDir(tmpFile, string(time.Now().UnixNano()))
```

Comment:
On my computer `path` is like: `/tmp/mesg-templates/�094045028`
There is a encoding error. You should `strconv.FormatInt(time.Now().UnixNano(), 10))` instead of `string(....)`

Reply:
even the `TempDir` accepts empty string for defaults and manages everything so I used that and it takes the defaults from that function better than trying to create a specific folder that might have some conflicts